pkg/operator/cluster/ceph/mon/health.go

```diff
@@ -126,6 +127,8 @@ func (c *Cluster) checkHealth() error {
 			if time.Since(c.monTimeoutList[mon.Name]) <= MonOutTimeout {
 				logger.Warningf("mon %s NOT found in quorum, STILL in mon out timeout", mon.Name)
 				continue
+			} else {
+				logger.Warningf("mon %s NOT found in quorum, timeout exceeded failover of mon", mon.Name)
```

Comment:
the wording here is a bit awkward, consider the following instead (but it's not a big deal):
```
mon %s NOT found in quorum and timeout has been exceeded, mon will be failed over.
```